{ 'cebit_greet_w_head': { 'head': [ { 'duration': 1.96,
                                      'name': u'cebit_greet_open',
                                      'positions': [-1.0678, -0.0603],
                                      'starttime': 0.0},
                                    { 'duration': 2.024,
                                      'name': u'cebit_greet_closed',
                                      'positions': [-0.9417, 0.3818],
                                      'starttime': 1.96},
                                    { 'duration': 2.007,
                                      'name': u'cebit_greet_open',
                                      'positions': [-1.0678, -0.0603],
                                      'starttime': 3.984},
                                    { 'duration': 2.007,
                                      'name': u'cebit_greet_closed',
                                      'positions': [-0.9399, 0.1884],
                                      'starttime': 5.991},
                                    { 'duration': 2.007,
                                      'name': u'ready_state',
                                      'positions': [-0.0, 0.0022],
                                      'starttime': 7.997999999999999}],
                          'left_arm': [],
                          'left_leg': [],
                          'right_arm': [ { 'duration': 2.024,
                                           'name': u'cebit_greet_open',
                                           'positions': [ 1.1606,
                                                          1.3987,
                                                          1.0485,
                                                          -0.0689,
                                                          0.5998,
                                                          0.2598,
                                                          -1.56],
                                           'starttime': 0.0},
                                         { 'duration': 1.96,
                                           'name': u'cebit_greet_closed',
                                           'positions': [ 2.2849,
                                                          0.9283,
                                                          1.3037,
                                                          -0.1707,
                                                          0.6004,
                                                          0.2592,
                                                          -1.56],
                                           'starttime': 2.024},
                                         { 'duration': 2.007,
                                           'name': u'cebit_greet_open',
                                           'positions': [ 1.1606,
                                                          1.3987,
                                                          1.0485,
                                                          -0.0689,
                                                          0.5998,
                                                          0.2598,
                                                          -1.56],
                                           'starttime': 3.984},
                                         { 'duration': 2.007,
                                           'name': u'cebit_greet_closed',
                                           'positions': [ 2.2849,
                                                          0.9283,
                                                          1.3037,
                                                          -0.1707,
                                                          0.6004,
                                                          0.2592,
                                                          -1.56],
                                           'starttime': 5.991},
                                         { 'duration': 2.007,
                                           'name': u'ready_state',
                                           'positions': [ 1.5135,
                                                          -0.7532,
                                                          0.2755,
                                                          -0.0212,
                                                          -0.0208,
                                                          0.2499,
                                                          -0.8168],
                                           'starttime': 7.997999999999999}],
                          'right_leg': [],
                          'torso': []}}
